core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/factory.go

```diff
@@ -95,12 +95,12 @@ func (o *LogTriggersOptions) defaultBlockRate() uint32 {
 func (o *LogTriggersOptions) defaultLogLimit() uint32 {
 	switch o.chainID.Int64() {
 	case 42161, 421613, 421614: // Arbitrum
-		return 1
+		return 2
```

Comment:
i think we can keep this as 1 and blockRate as 2? Would be clearer to say we'll do 1 log per 2 blocks instead of 2 logs per 4 blocks which will have slightly different meaning for log selection